specification.markdown

```diff
@@ -320,7 +320,7 @@ The path defines the allowed storage locations of the `EBML Element` within an `
 The `path` attribute is REQUIRED.
 
 ```
-EBMLFullPath             = EBMLElementOccurrence "(" EBMLReferencePath ")"
+EBMLFullPath           = EBMLElementOccurrence "(" EBMLReferencePath ")"
```

Comment:
Could also resolve by renaming `VariableParentOccurrence` to `VarParentOccurrence`.